aslprep/tests/data/license.txt

```diff
@@ -0,0 +1,4 @@
+stauffer@upenn.edu
```

Comment:
We probably don't want this on github.

Reply:
Good point. How do you handle the license file for testing purposes in qsiprep?

Reply:
Hopefully this is fixed in [c74cd01](https://github.com/PennLINC/aslprep/pull/204/commits/c74cd01e2849acbe3663ffaa043bd31ff918c4d7) and [cc43264](https://github.com/PennLINC/aslprep/pull/204/commits/cc43264dc8aa7c9c4765face01c2bbf59eca4100).